yew/src/callback.rs

```diff
@@ -12,26 +30,46 @@ use std::rc::Rc;
 /// </aside>
 /// An `Rc` wrapper is used to make it cloneable.
 pub enum Callback<IN> {
-    /// A callback which can be called multiple times
-    Callback(Rc<dyn Fn(IN)>),
+    /// A callback which can be called multiple times with optional modifier flags
+    Callback {
+        /// A callback which can be called multiple times
+        cb: Rc<dyn Fn(IN)>,
+
+        /// Sets flags for event listening. A combination of `PASSIVE`, and `HANDLE_BUBBLED`.
+        ///
+        /// If None, the default flags for the callback event source are used.
+        ///
+        /// `DEFER` implies `PASSIVE`.
+        /// `DEBOUNCE` implies `PASSIVE` and `DEFER`.
+        ///
+        /// Currently only used with `feature = "web_sys"`.
+        flags: Option<u8>,
+    },
+
     /// A callback which can only be called once. The callback will panic if it is
     /// called more than once.
-    CallbackOnce(Rc<CallbackOnce<IN>>),
+    Once(Rc<Once<IN>>),
 }
 
-type CallbackOnce<IN> = RefCell<Option<Box<dyn FnOnce(IN)>>>;
+type Once<IN> = RefCell<Option<Box<dyn FnOnce(IN)>>>;
```

Comment:
`Once` isn't a very descriptive name for this type. How about `DynamicFnOnce` to convey the idea that it is a `FnOnce` but handled at runtime?

Reply:
Might be too verbose.Would like @jstarry's opinion on this.

Reply:
@bakape @siku2 I don't care too much but I'd prefer to keep this name change out of this PR, it doesn't seem necessary to update right now